test/test.js

```diff
@@ -92,7 +91,6 @@ describe('Test coloredcoinsd', function () {
     }
     cc.broadcastTx(args, function (err, ans) {
       assert(err)
```

Comment:
It does return an error, but not the specified expected error.
I will open a separate issue so such an error will be returned again (it does not relate to my changes).